conda/0_sortedcontainers/build.sh

```diff
@@ -1,9 +0,0 @@
-#!/bin/bash
-
-$PYTHON setup.py install
-
-# Add more build steps here, if they are necessary.
-
-# See
-# http://docs.continuum.io/conda/build.html
-# for a list of environment variables that are set during the build process.
```

Comment:
TIL you can move this into the `.yaml` files. That's cool!

Reply:
Yep, that's from the newer version of conda-build. I used `conda skeleton` to re-generate these recipes.